bin/score.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+
+import json
+import sys
+
+
+def score_submission(predictions_path: str, status: str) -> dict:
+    """Determine the score of a submission. This is a placeholder function.
+
+    Args:
+        predictions_path (str): path to the predictions file
+        status (str): current submission status
+
+    Returns:
+        result (dict): dictionary containing score, status and errors
+    """
+    if status == "INVALID":
+        score_status = "INVALID"
+        score = None
+    else:
+        # placeholder file reading
+        with open(predictions_path, "r") as sub_file:
+            predictions_contents = sub_file.read()
+        try:
+            # placeholder scoring
+            score = 1 + 1
+            score_status = "SCORED"
+            message = ""
+        except Exception as e:
+            message = f"Error {e} occurred while scoring"
+            score = None
+            score_status = "INVALID"
+    result = {"score": {"auc": score, "status": score_status, "errors": message}}
+    return score_status, result
+
+
+def update_json(results_path: str, result: dict) -> None:
+    """Update the results.json file with the current score and status
+
+    Args:
+        results_path (str): path to the results.json file
+        result (dict): dictionary containing score, status and errors
+    """
+    with open(results_path, "r") as o:
+        data = json.load(o)
+    data.update(result)
+    with open(results_path, "w") as o:
+        o.write(json.dumps(data))
+
+
+if __name__ == "__main__":
+    predictions_path = sys.argv[1]
+    results_path = sys.argv[2]
```

Comment:
What is the rationale for passing in the existing results.json when you can create a new one?

Reply:
My reasoning was that I wanted to have each step that writes to `results.json` add to the record so that at the end you would have a complete `results.json` to look at for information on each step. 

If this is not desirable, I can make that change and it will remove the need to read the file which opens us up to the empty file error